libcodechecker/libhandlers/store.py

```diff
@@ -18,11 +17,8 @@
 import tempfile
 import traceback
 
-import shared
-
 from libcodechecker import generic_package_context
 from libcodechecker import host_check
-from libcodechecker import suppress_file_handler
```

Comment:
We removed the import of this lib. I take it it's still used elsewhere in the `cmd` so we can't fully get rid of it? (Yet?)